frontend/src/features/overviewpage/OverviewPageContent.tsx

```diff
@@ -1,103 +1,41 @@
 
 export const OverviewPageContent = () => {
-  // Fix-me: CollectionBar links go nowhere yet
+  const { data: systemUsers, isError: isLoadSystemUsersError } = useGetSystemUsersQuery();
+  const { data: rights, isError: isLoadRightsError } = useGetRightsQuery();
 
   const dispatch = useAppDispatch();
-  const reduxObjektArray = useAppSelector((state) => state.overviewPage.systemUserArray);
-  const postConfirmed = useAppSelector((state) => state.creationPage.postConfirmed);
-
-  useEffect(() => {
-    // If user reverts to OverviewPage after New SystemUser creation,
-    // using BrowserBackButton,
-    // we need to reset postConfirmed and do fetchOverviewPage here
-    if (postConfirmed) {
-      void dispatch(resetPostConfirmation());
-      void dispatch(fetchOverviewPage());
-    }
-  }, [dispatch, postConfirmed]);
-
-  const { t } = useTranslation('common'); // not used yet
+  const { t } = useTranslation();
   const navigate = useNavigate();
 
-  const rightsObjektArray = useAppSelector(
-    (state) => state.rightsIncludedPage.systemRegisterProductsArray,
-  );
-  const compact: boolean = false; // not used yet
-
-  // INNERMOST LAYER of RightCollectionBar-inside-SystemUserCollectionBar setup
-  // mock array for ActionTags such as Read, Write, Sign etc:
-  // *********API not ready************
-  // but should be flexible for future new actions, such as Rune´s Launch-Rocket
-  const mockRightsActionsArray = [
-    { name: 'Lese', on: true },
-    { name: 'Skrive', on: false },
-    { name: 'Signere', on: true },
-    { name: 'Les arkiv', on: false },
-    { name: 'Launch-Rune´s-Rocket', on: true },
-  ];
-
-  // The Tags are mapped out of the mockRightsActionsArray
-  const onlyTags = mockRightsActionsArray.map((mockRightsActions, index) => (
-    <div key={index} className={classes.tagSeparator}>
-      <Tag size='small' color={mockRightsActions.on ? 'info' : 'danger'}>
-        {mockRightsActions.name}
-      </Tag>
-    </div>
-  ));
-
-  // MIDDLE LAYER of RightCollectionBar-inside-SystemUserCollectionBar setup
-  // consumes Tag-array as collection
-  // uses custom CollectionBar InnerCollectionBar
-  // Fix-me: colors very restricted via base "class" ActionBar... should port
-  // light blue such as "tertiary", "third" or something later
-  const currentRightsCollectionBars = rightsObjektArray.map((ProductRight, index) => (
-    <div key={index}>
-      <ActionBar
-        title={ProductRight.right}
-        subtitle={ProductRight.serviceProvider}
-        color='neutral'
-        actions={
-          <Button variant='tertiary' size='small'>
-            Fjern rettighet
-            <MinusCircleIcon />
-          </Button>
-        }
-      >
-        <div>
-          <p>Eventuell tekst om rettighetene kommer her.</p>
-          <div className={classes.rightActionTagsWrapper}>{onlyTags}</div>
-        </div>
-      </ActionBar>
-    </div>
-  ));
+  useEffect(() => {
```

Comment:
I recommend creating a hook that better describes the intent here. Something like `useFirstRenderEffect`, `useOnceEffect` (naming is hard). `useEffect` has too many use-cases, and someone who's new might not know that this effect will only run once - unless the identity of the `dispatch` function changes in which case you're in for some real pain. And even in that case, it's important to know that this effect will be ran more than once in certain situations (like debugging).